atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -154,14 +154,9 @@ private long waitForSchemaMutationLock() {
                     int mutationTimeoutMillis = configManager.getConfig().schemaMutationTimeoutMillis();
                     // possibly dead remote locker
                     if (stopwatch.elapsed(TimeUnit.MILLISECONDS) > mutationTimeoutMillis * 4) {
-                        throw new TimeoutException(String.format("We have timed out waiting on the current"
-                                + " schema mutation lock holder. We have tried to grab the lock for %d milliseconds"
-                                + " unsuccessfully.  Please try restarting the AtlasDB client. If this occurs"
```

Comment:
I think part of the flag I raised around severity is the message itself -- the warning this ultimately generates says "restart your servers" -- if restarting isn't necessary because servers will try again, that's fine, but then the message is broken. 

Besides, we did actually observe a lock associated with a dead server.

Maybe the real fix here is to heartbeat the lock so we can detect staleness and provide better messages?

Crux of my complaint is that you shouldn't log a warning that says something is fatal--we can make the thing fatal, but if it's not actually a problem then maybe we should just fix the warning. But I can't tell from the commit comments if this is actually fatal...